include/ck_tile/host/arg_parser.hpp

```diff
@@ -148,18 +197,21 @@ class ArgParser
 
     int get_int(const std::string& name) const
     {
+        assert(("The param contians invalid number", ValidNumber::isNumber(get_str(name)) == true));
```

Comment:
Let's not be so pedantic. This is redundant. And moreover the `assert` function is no-op in Release build so it won't be checked anyway.

Reply:
I believe that no safeguard is pedantic and redundant. It simply protects us from any errors. Yes, I thought which mode. 

Indeed, it only works in Debug mode. I was wondering in which mode use to throw the error. Unfortunately, we don't have a logging structure, so I decided to go with Debug.